### DISCUSSION TITLE
Drop uid checking

### DISCUSSION THREAD

**Discussion by sunwire** [scsitape/stenc#64] [Ideas]
Can we drop checking if the user is root?
https://github.com/scsitape/stenc/blob/ebabc39c3eb57f8dba3921fe7909ada06eb8d734/src/main.cpp#L228-L230
Now when we use a syslog for logs, at least under Linux, an ordinary user can use Stenc if he has the right to read the tape device and Stenc has  set  cap_sys_rawio capability.
https://man7.org/linux/man-pages/man7/capabilities.7.html
https://man7.org/linux/man-pages/man8/setcap.8.html

**Comment by jonasstein** [comment]
Dropping sounds good to me. 
Could we then use the `tape` group too? We should add a section on permissions to the man page.

**Comment by jmwilson** [comment]
Yes, this is dumb to check. Would be good to see how the program fails with a nonprivileged user so the error message is not totally inscrutable.
